node/node.go

```diff
@@ -168,6 +175,29 @@ func Setup(ctx context.Context, init NodeInitializer) (*Node, error) {
 	}
 
 	h, err := libp2p.New(opts...)
+
+	//	peering service
+	hPeer := peering.NewEstuaryPeeringService(h)
+
+	peeringPeerList := cfg.PeeringPeers
+	peeringPeerList = append(peeringPeerList, bootStrapPeeringPeers...)
```

Comment:
nit: condense to `peeringPeerList := append(cfg.PeeringPeers, bootStrapPeeringPeers...)`